examples/configs/src/view-configs/blin.js

```diff
@@ -13,9 +13,20 @@ export const blin2019 = {
       name: 'idr0062-blin-nuclearsegmentation/6001240',
       files: [
         {
-          type: 'raster',
-          fileType: 'raster.ome-zarr',
+          fileType: 'image.ome-zarr',
           url: 'https://minio-dev.openmicroscopy.org/idr/v0.3/idr0062-blin-nuclearsegmentation/6001240.zarr',
+          options: {
+            coordinateTransformations: [
+              {
+                type: 'translation',
+                translation: [1, 1],
```

Comment:
Also according to the spec shouldn't this be the same length as the dimensions of the image? This doesn't match the `translation`

Reply:
Should we throw an error in this case then, enforcing the spec, or assume the final value as 0/1 (maybe with warning in console)?

Reply:
i would lean towards enforcing the spec since it is the spec